aperturedb/BBoxLoader.py

```diff
@@ -10,21 +10,22 @@
 
 
 class BBoxGeneratorCSV(CSVParser.CSVParser):
-    """**ApertureDB BBox Data loader.**
+    """**ApertureDB Bounding Box Data generator.**
 
     .. note::
-        Expects a csv file with the following columns:
+        Is backed by a csv file with the following columns:
 
             ``IMG_KEY``, ``x_pos``, ``y_pos``, ``width``, ``height``, ``BBOX_PROP_NAME_1``, ... ``BBOX_PROP_NAME_N``
```

Comment:
this (and any CSV generator, I believe) can also have "constraints_PROP" to specify conditional adding. Would you add that? And maybe an explanation in CSVGenerator about how that is used.